package.json

```diff
@@ -7,7 +7,8 @@
     "scripts": {
         "build": "node utils/build.js",
         "lint": "eslint src/js",
-        "package": "utils/package.sh"
+        "package": "utils/package.sh",
+        "winpackage": "utils\\winpackage.cmd"
```

Comment:
I think the escaped backslash will break unix like builds - whereas windows is tolerant of forward slashes in this case.